### PR TITLE
Evalcast

### PR DESCRIPTION
**Pull request by dajmcdon**
@ryantibs @capnrefsmmat 

It's been many moons since the old `evalcast` went to main.

* This is current "production" `evalcast`. I bumped the version number.
* It adds the pkgdown site in the `docs/` directory.
* I just merged in `main` so hopefully there's not too much iteration going on there.
* Please advise if I screwed something up royally (or just used poor gitflow).

### CODE REVIEW

File: R-packages/evalcast/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+```r
+remotes::install_github("cmu-delphi/covidcast", ref = "main",
+                         subdir = "R-packages/evalcast")
+```
+
+## Documentation
+
+The package documentation and examples are [available
+online](https://cmu-delphi.github.io/covidcast/evalcast/index.html).
```
Comment (capnrefsmmat) [inline R-packages/evalcast/README.md:29]:
```suggestion
online](https://cmu-delphi.github.io/covidcast/evalcastR/index.html).
```

File: R-packages/evalcast/vignettes/intro-evalcast.Rmd [hunk 2]
```diff
@@ -0,0 +1,211 @@
+---
+title: "Introduction to evalcast"
```
Comment (capnrefsmmat) [inline R-packages/evalcast/vignettes/intro-evalcast.Rmd:2]: If you name the vignette the same thing as the package, pkgdown automatically makes a "get started" link in the top menu for that vignette. Your decision if that's nicer: https://pkgdown.r-lib.org/reference/build_articles.html#get-started

**Review by capnrefsmmat — CHANGES_REQUESTED** [review]
Overall looks fine -- just one incorrect link and one thing you can change if you want

File: R-packages/evalcast/vignettes/intro-evalcast.Rmd (hunk 2 above)
Comment (dajmcdon) [reply R-packages/evalcast/vignettes/intro-evalcast.Rmd]: Thanks! Done.

**Comment by ryantibs** [comment]
Wow this looks like a lot!  Since Alex reviewed the "scaffolding" and not the code, I guess that leaves me to review the code???

At this point, given that we (Delphi) are very likely the only ones using this package, I am fine to merge to main with the knowledge that this is code is functional-enough since you guys have been using it on a branch for a while.

**Comment by dajmcdon** [comment]
@ryantibs I don't think you need to review the code. It's mostly been reviewed ages ago. Just so one or both of you makes sure it doesn't hurt anything else in the repo.

**Comment by capnrefsmmat** [comment]
Did you fully rebuild the pkgdown site and commit it? I still see the index page pointing to the wrong link to the documentation (evalcast instead of evalcastR)

**Comment by dajmcdon** [comment]
Ugh. Rebuilt and then merged your suggested commit rather than the other way.

**Review by krivard — APPROVED** [review]
CI and .gitignore look good

**Review by capnrefsmmat — APPROVED** [review]
Looks good to merge to me. We can check the rendered documentation site afterward and fix any issues that might come up

**Comment by dajmcdon** [comment]
Great! Please go ahead and merge (I don't have permission).

**Comment by capnrefsmmat** [comment]
by the powers vested in me, I hereby dub thee "merged"

**Pull request merged**